ui/src/app/pages/data/concept/concept-homepage.spec.tsx

```diff
@@ -44,7 +44,6 @@ describe('ConceptHomepage', () => {
     serverConfigStore.set({
       config: {
         ...defaultServerConfig,
-        enableDrugWildcardSearch: false,
       },
     });
```

Comment:
Remove the entire `serverConfigStore` reference.

Reply:
done